content/rust/concepts/comments/comments.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Comments'
+Description: 'A comment is a a line, or a few lines, of text which add an extra explanation to the code to ease readability of the code for people.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Best Practices'
+  - 'Comments'
+  - 'Documentation'
+CatalogContent:
+  - 'rust-for-programmers'
+  - 'paths/computer-science'
+---
+
+A **comment** is a line, or few lines, of text which add extra explanation to the code to ease readability of the code for people. Any comment will be ignored and not executed by the compiler. In Rust programming, there are two types of comments:
```

Comment:
```suggestion
A **comment** is a line, or few lines, of text that adds extra explanation to the code to ease readability of the code for people. Any comment will be ignored and not executed by the compiler. In Rust programming, there are two types of comments:
```

Reply:
Hi, thanks. Changes done.

Reply:
Do I need to open a new PR?

Reply:
Nope, there's no need for a new PR. We usually suggest changes if required; it is expected that you make them in the same branch and PR.